src/lib.rs

```diff
@@ -44,21 +45,7 @@ pub async fn run() {
     tidy_algo.load_rules_from_file(&my_files, PathBuf::from("config/rules/basic.yml"));
```

Comment:
That's my bad, I should have modified it in the first place
```suggestion
    tidy_algo.load_rules_from_file(&my_files, PathBuf::from(vec![r"config", r"rules", r"basic.yml"].iter().collect()));
```